CONTRIBUTING.md

```diff
@@ -150,13 +150,13 @@ All pull requests are reviewed by another person. We have a bot,
 request.
 
 If you want to request that a specific person reviews your pull request,
-you can add an `r?` to the message. For example, [Steve][steveklabnik] usually reviews
+you can add an `r?` to the pull request message. For example, [Steve][steveklabnik] usually reviews
 documentation changes. So if you were to make a documentation change, add
 
     r? @steveklabnik
 
-to the end of the message, and @rust-highfive will assign [@steveklabnik][steveklabnik] instead
-of a random person. This is entirely optional.
+to the end of the pull request message, and [@rust-highfive][rust-highfive] will assign
```

Comment:
The change from `@rust-highfive` to `[@rust-highfive][rust-highfive]` could very well be undesirable. I did that because the non-highlighted [at] text looked weird. Feel free to revert it!

```suggestion
to the end of the pull request message, and @rust-highfive will assign
```

Reply:
I'm good with your change